app/org/maproulette/models/dal/ChallengeDAL.scala

```diff
@@ -729,25 +729,34 @@ class ChallengeDAL @Inject()(override val db: Database, taskDAL: TaskDAL,
                    FROM ( SELECT 'Feature' As type,
                                   ST_AsGeoJSON(lg.geom)::json As geometry,
                                   hstore_to_json(lg.properties)::jsonb ||
-                                  json_build_object('maproulette',
+
```

Comment:
This may just be a matter of me not quite seeing what is going on here. But where did the json_build_object function go to exactly? Or do we just not need it anymore? I can't remember the specifics of this sql so apologize if this is a stupid question.